cirq-core/cirq/sim/act_on_state_vector_args.py

```diff
@@ -108,9 +399,8 @@ def swap_target_tensor_for(self, new_target_tensor: np.ndarray):
             new_target_tensor: The new system state. Must have the same shape
                 and dtype as the old system state.
         """
-        if new_target_tensor is self.available_buffer:
-            self.available_buffer = self.target_tensor
-        self.target_tensor = new_target_tensor
+        # TODO: deprecate this function, it is unused and should not have been made public.
```

Comment:
Hanging TODO? If this is not meant to be resolved here, please open an issue to follow up.